flink-tests/src/test/java/org/apache/flink/test/classloading/ClassLoaderITCase.java

```diff
@@ -137,6 +137,20 @@ public void testJobsWithCustomClassLoader() throws IOException, ProgramInvocatio
 
 			inputSplitTestProg.invokeInteractiveModeForExecution();
 
+		} catch (Exception e) {
+			if (!(e.getCause() != null &&
```

Comment:
please at least wrap this `if` statement into a function `swallowScuessException(e)` or something like that.

Another idea is to maybe use this pattern:
```
	@Rule
	public ExpectedException expectedException = ExpectedException.none();

	@Test
	public void test() {
		expectedException.expectCause(isA(SuccessException.class));
		somethingThatThrows();
	}
```
( https://stackoverflow.com/a/20759785/8149051 )